service-discovery/src/main/resources/application.yml

```
@@ -4,8 +4,15 @@ server:
+    port: 5672
+    username: guest
+    password: guest
 
 eureka:
   client:
     register-with-eureka: false
-    fetch-registry: false
\ No newline at end of file
+    fetch-registry: false
+  server:
+    wait-time-in-ms-when-sync-empty: 15
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)